core/app/services/spree/variants/remove_line_items.rb

```diff
@@ -0,0 +1,17 @@
+module Spree
+  module Variants
+    class RemoveLineItems
+      prepend Spree::ServiceModule::Base
+
+      def call(variant:)
+        cart_remove_item_service = Spree::Dependencies.cart_remove_item_service.constantize
+        incomplete_orders = variant.orders.where.not(state: 'complete')
+        incomplete_orders.each do |order|
+          cart_remove_item_service.call(variant: variant, order: order)
```

Comment:
@aleksandarpetrushev this is easy to get errored (DB timeout, etc) - we should enqueue this into it's own job so these line items can be removed independently and safely

Reply:
@damianlegawiec you mean remove every line item in a separate job?

Reply:
@aleksandarpetrushev yup